### PR TITLE
refact(skymp5-client): improve browser interaction

### PR DESCRIPTION
**Pull request by alekcey0211**
add enums for KeyCodes from https://www.creationkit.com/index.php?title=Input_Script and ui menu names, add simple event-emitter implementation, rewrite browser interaction with this feature

### CODE REVIEW

File: skymp5-client/src/front/browser.ts [hunk 1]
```diff
@@ -1,93 +1,102 @@
 import {
-  browser,
-  on,
-  once,
-  Input,
-  printConsole,
-  settings,
-  Ui,
+	browser,
```
Comment (nic11) [inline skymp5-client/src/front/browser.ts:2]: Pls return spaces here

File: skymp5-client/src/lib/ui-menu.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	Training = 'Training Menu',
+	Tutorial = 'Tutorial Menu',
+	Tween = 'TweenMenu',
+}
+
+export const badMenus: UiMenu[] = [
+	UiMenu.Barter,
+	UiMenu.Book,
+	UiMenu.Container,
+	// UiMenu.Crafting,
```
Comment (nic11) [inline skymp5-client/src/lib/ui-menu.ts:44]: Is it commented on purpose?

**Comment by nic11** [comment]
Super quick look

File: skymp5-client/src/lib/ui-menu.ts (hunk 2 above)
Comment (alekcey0211) [reply skymp5-client/src/lib/ui-menu.ts]: uncomment this.

File: skymp5-client/src/front/browser.ts (hunk 1 above)
Comment (alekcey0211) [reply skymp5-client/src/front/browser.ts]: ok, but i prefer 2 sizes tabs. And it seems to me that it is more correct.

**Comment by alekcey0211** [comment]
@Pospelove review plz.

File: skymp5-client/src/front/browser.ts [hunk 3]
```diff
@@ -7,76 +7,85 @@ import {
   settings,
```
Comment (Pospelove) [inline skymp5-client/src/front/browser.ts:7]: Code became less readable. Please find a way to configure bindings as you want without bloating up the code.

File: skymp5-client/src/front/browser.ts [hunk 4]
```diff
@@ -7,76 +7,85 @@ import {
...
+  const keyState: { num: number } = { num: 0 };
+
+  on('update', () => {
+    const numKeys = Input.getNumKeysPressed();
+
+    if (keyState.num !== numKeys) {
+      keyState.num = numKeys;
+      const keyCodes = Array(numKeys)
+        .fill(null)
+        .map((_, i) => Input.getNthKeyPressed(i));
```
Comment (Pospelove) [inline skymp5-client/src/front/browser.ts:49]: It seems to be too expensive to check the whole keyboard. Let's check only bound keyboard keys. Calling game functions is expensive.

**Review by Pospelove — CHANGES_REQUESTED** [review]

**Review by Pospelove — CHANGES_REQUESTED** [review]
Remove event emitter for now since it isn't used in your latest PR. We can go back to event emitter in a further pull request and add it to SP

**Pull request merged**